DISCLAIMER.md

```diff
@@ -0,0 +1,49 @@
+
+The license under which an icon is available can be found in the icon's entry in the JSON data file in the NPM and Packagist packages.
+
+As we rely largely on our community to help us keep Simple Icons up-to-date, the license data for an icon may be outdated. If you find that the license data provided is outdated, please [submit an issue] to [the Simple Icons GitHub repository].
+
+If an icon includes a registered trademark (`®`) or trademark symbol (`™`) the recommendations outlined in [the Simple Icons Contributing Guidelines] are followed to decide whether to include the symbol or not.
+
+Simple Icons cannot be held responsible for any legal activity raised by a brand, or users of the package. We ask that our users seek the correct permissions to use the icons relevant to their project.
+
+## Brand Guidelines
+
+> NOTE: the addition of guidelines to Simple Icons is an ongoing project. In the meantime, users of Simple Icons are instead encouraged to check the `source` URL as, in some cases, the icon will have been sourced from official guidelines. The lack of a `guidelines` entry for a particular brand does not imply that the brand has no guidelines.
+
+Simple icons provides a link to a brand's _branding guidelines_ (or similar) if the brand provides one. We ask our users read these guidelines and ensure their usage of the brand's icon is in accordance with them. As guidelines are subject to change we also ask our users to regularly check if the brand guidelines of the icons they use have been updated.
+
+The brand guidelines can be found in the icon's entry in the JSON data file in the NPM and Packagist packages, or through the "Brand Guidelines" links on [the Simple Icons website].
+
+As we rely largely on our community to help us keep Simple Icons up-to-date, the guidelines link for a brand may be outdated. If you find that the guidelines are outdated, please [submit an issue] to [the Simple Icons GitHub repository].
+
+## Update of Brands
+
+Should a brand wish for their icon or data to be updated, please [submit an issue] to [the Simple Icons GitHub repository]. In the issue, please explain your affiliation with the company and reasons for the update. We generally release updates once a week, but can occasionally make exceptions for immediate updates of brands.
+
+## Removal of Brands
+
+Should a brand wish for their icon to be removed from the package, contact `removals at simpleicons dot org` explaining your affiliation with the company, and reasons for removal. Alternatively, it is also possible to [submit an issue] on [the Simple Icons GitHub repository] with the same information. We generally remove icons that no longer [meet our criteria](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#requesting-an-icon) twice a year in our major releases - but can occasionally make exceptions for immediate removal of brands.
+
+[submit an issue]: https://github.com/simple-icons/simple-icons/issues/new/choose
```

Comment:
Since all "submit an issue" links fit within the scope of the `icon outdated` label, why not just pointing to that template opening issues?

```suggestion
[submit an issue]: https://github.com/simple-icons/simple-icons/issues/new?labels=icon+outdated&template=icon_update.md
```

Reply:
Though I'm fine either way, the current URL is less likely to change in the future so I'd be slightly in favor of that.